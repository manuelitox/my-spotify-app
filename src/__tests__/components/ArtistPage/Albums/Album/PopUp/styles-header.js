import React from 'react'
import { shallow } from 'enzyme'

import StylesHeaderAlbum from 'components/ArtistPage/Albums/Album/PopUp/styles-header'

describe('Component | StylesHeaderAlbum', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StylesHeaderAlbum />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})