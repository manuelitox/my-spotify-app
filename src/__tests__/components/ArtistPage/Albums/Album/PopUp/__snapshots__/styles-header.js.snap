// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StylesHeaderAlbum should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  margin: 30px 0;
  img {
    @media only screen and (max-width: ",
          [Function],
          ") {
      height: auto;
      max-width: 300px;
      width: 100%;
    }
  }
  h3 {
    color: ",
          [Function],
          ";
    font-size: 1.6rem;
    font-weight: 500;
    margin: 10px 0 4px;
  }
  p {
    color: ",
          [Function],
          ";
    font-size: 1.4rem;
    line-height: 1.6;
    margin: 0;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
